src/components/bankAccountPropTypes.js

```diff
@@ -2,14 +2,14 @@ import PropTypes from 'prop-types';
 
 export default PropTypes.shape({
     /** The name of the institution (bank of america, etc */
-    addressName: PropTypes.string,
+    title: PropTypes.string,
 
     /** The masked bank account number */
-    accountNumber: PropTypes.string,
+    description: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
 
     /** The bankAccountID in the bankAccounts db */
-    bankAccountID: PropTypes.number,
+    methodID: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
 
     /** The bank account type */
-    type: PropTypes.string,
+    accountType: PropTypes.string,
```

Comment:
This change will affect every component that uses bankAccountPropTypes, I don't think we should update this.

Reply:
+1, we don't want to do this